verification/curator-service/api/src/index.ts

```diff
@@ -225,6 +225,11 @@ new OpenApiValidator({
             mustHaveAnyRole(['reader', 'curator', 'admin']),
             casesController.list,
         );
+        apiRouter.get(
+            '/cases/download',
+            mustHaveAnyRole(['reader', 'curator', 'admin']),
```

Comment:
not sure if it makes sense for admins to be able to download

Reply:
I chose it because it matched `/cases` above. Figured if they could query for all the cases then they could download all the cases. But I don't feel strongly here - I could change one or both?

Reply:
ah right I added that because of the no-limit delete functionality I added recently. Fine as is then makes sense to be able to download if they can read indeed.